### PR TITLE
Add Project.toml

### PR DESCRIPTION
**Pull request by cmcaine**
Pkg.dev on Julia 1.6 was giving me an error without one. I couldn't
easily test the package, either.

### CODE REVIEW

**Comment by cmcaine** [comment]
The tests fail locally for me, but they failed before I made this change, too, so idk.

Doing this partly so I can try to close https://github.com/JuliaWeb/Mux.jl#113.

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+[deps]
+Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
+Sockets = "6462fe0b-24de-5631-8697-dd941f90decc"
+
+[compat]
+HTTP = "0.8"
+julia = "0.7, 1"
```
Comment (hustf) [inline Project.toml:13]: I suppose "0.7, 1" includes all Julia versions up to 2.0.

**Comment by hustf** [comment]
Thanks! I don't see why the testing setup can't find Test. The test is going to fail anyway due to some minor changes in the printing format.  I'm merging this to make further testing easier.

This package is in need of some maintenance, but I believe it's of value as a backup for HTTP.Websockets, as that package is doing some experimentation now.

**Comment by christopher-dG** [comment]
You need to add:

```toml
[extras]
Test = "TEST-UUID-HERE"

[targets]
test = ["Test"]
```

And then Test will be picked up as a test-only dependency.

**Comment by hustf** [comment]
@cmcaine, would you have the change to follow up Christopher's helpful comment with another PR?
Locally, it's working out of the box of course.

**Comment by cmcaine** [comment]
On Julia 1.6 I didn't need those lines, but I'll add them and test on whatever older Julia I have installed :)

I wonder if we're actually still compatible with Julia 0.7? Do you care about maintaining compat with that?

**Comment by hustf** [comment]
I just tested, too, and we can say WebSockets is working on Julia 0.4.7. Although that's version v0.3 that is being downloaded, nobody's left in the lurch by dropping support of old Julia versions.

Julia 0.7 version was the hardest one to support well due to all the deprecation messages. After dropping it from future versions, I believe some alternative tests could be dropped. But the important fix will be to update 'travis.yml' and 'appveyor.yml'.

In my experience, getting things to work on those testing services is a lot more difficult than having tests work locally, so be warned. If you work in a pull request that you update, I believe you can read the online test logs and possibly retrigger the tests in case you suspect a failure was just a fluke.

**Pull request merged**